libretroshare/src/gxs/rsgxsrequesttypes.h

```diff
@@ -36,18 +36,23 @@ struct GxsRequest
 	uint32_t token;
 	uint32_t reqTime;
 
-	RS_DEPRECATED uint32_t ansType; /// G10h4ck: This is of no use
+	RS_DEPRECATED uint32_t ansType; /// G10h4ck: This is of no use. csoler: it's made available to the clients.
```

Comment:
For what it is used?

Reply:
This is made available to clients to have their own subtype for GxsRequests. 
This can be used for instance in loadRequest() later on.

Reply:
Ok now I get for what it can be useful, can you add some doxygen about it explaining how to use it? In that case you can also remove de deprecation. Maybe another name could be better... I don't remember to see this used much so it probably won't cause a refactor chain reaction

Reply:
ok. Good idea.

Reply:
a the same time that structure is not part of the API, so there's no doxygen comment block here.

Reply:
in the end I changed the variable name to something more explicit and improved the comment.